.github/workflows/scripts/start-firestore-emulator.sh

```diff
@@ -4,11 +4,14 @@ if ! [ -x "$(command -v firebase)" ]; then
   exit 1
 fi
 
-firebase emulators:start --only firestore &
-
-until curl --output /dev/null --silent --fail http://localhost:8080; do
-  echo "Waiting for Firestore emulator to come online..."
-  sleep 2
-done
-
-echo "Firestore emulator is online!"
\ No newline at end of file
+IS_CI="${CI}${CONTINUOUS_INTEGRATION}${BUILD_NUMBER}${RUN_ID}"
```

Comment:
Nice, I had this hacked in my local tree as I was bringing up my env, don't need to propose it now.
Any reason not to have a unified emulator script though? I don't believe it affects startup time (was unnoticeable to me) and it's easier to maintain one script. In RNFB I just start with an '--only firestore,auth' now and it's fine